pkg/apis/bundle/v1alpha1/patch_template_types.go

```diff
@@ -19,6 +19,30 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// ObjectSelector is used for identifying Objects on which to apply the patch template.
+type ObjectSelector struct {
+        // Kinds represent the Kinds to filter on.
+        Kinds []string
+
+        // Names represent the names to filter on. For objects, this is the
+        // metadata.name field. For components, this is the ComponentName.
+        Names []string
+
+        // Annotations contain key/value pairs to filter on. An empty string value matches
+        // all annotation-values for a particular key.
+        Annotations map[string]string
+
+        // Labels contain key/value pairs to filter on. An empty string value matches
+        // all label-values for a particular key.
+        Labels map[string]string
+
+        // Namespaces to filter on.
+        Namespaces []string
+
+        // Invert the match
```

Comment:
By default does this match things or filter things? I assume the former; would be good to call out explicitly